packages/tracing/src/Tracer.ts

```diff
@@ -261,7 +261,9 @@ class Tracer implements TracerInterface {
       return this.provider.captureAWSClient(service);
     } catch (error) {
       try {
-        return this.provider.captureAWSClient((service as unknown as T & { service: T }).service);
+        this.provider.captureAWSClient((service as unknown as T & { service: T }).service);
```

Comment:
can we add a comment to explain why we have to do this workaround ?

Reply:
Will do both of your comments

Reply:
Added comment to explain + link to issue in commit [9e52277](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/528/commits/9e5227713035881153b410151dc95d618febf149)!